adapter/pkg/config/parser.go

```diff
@@ -66,25 +88,28 @@ func GetMgwHome() string {
 // from where the executable is called from.
 //
 // Returns the log configuration object mapped from the configuration file during the startup.
-func ReadLogConfigs() (*LogConfig, error) {
+func ReadLogConfigs() *LogConfig {
 	onceLogConfigRead.Do(func() {
-		adapterLogConfig = new(LogConfig)
-		_, err := os.Stat(GetMgwHome() + relativeLogConfigPath)
+		adapterLogConfig = getDefaultLogConfig()
```

Comment:
Couldn't find the local function for getDefaultLogConfig()

Reply:
The function is moved to log_types.go file. Since both files are located within the same package, it is resolved.